service/src/main.rs

```diff
@@ -58,7 +58,7 @@ use its_primitives::types::SignedBlock as SignedSidechainBlock;
 use log::*;
 use my_node_runtime::{pallet_teerex::ShardIdentifier, Event, Hash, Header};
 use sgx_types::*;
-use sidechain_storage::{BlockPruner, SidechainStorageLock};
+use sidechain_storage::{self, BlockPruner, SidechainStorageLock};
```

Comment:
what's this self for?

Reply:
for start_sidechain_pruning_loop. You prefer to import the method directly?

Reply:
I am confused. As far as I read the code, this should not be necessary the way you call the method.

Regardless, you can also import the function directly to be less verbose at the calling site.

Reply:
👍 for importing the function, not `self`.